src/AutoRest.CSharp.V3/Properties/launchSettings.json

```diff
@@ -24,6 +24,10 @@
       "commandName": "Project",
       "commandLineArgs": "--standalone $(SolutionDir)\\samples\\Azure.Network.Management.Interface\\Azure.Network.Management.Interface"
     },
+    "Azure.Storage.Blobs": {
+      "commandName": "Project",
+      "commandLineArgs": "--standalone $(SolutionDir)\\samples\\Azure.Storage.Blobs\\Azure.Storage.Blobs"
+    },
```

Comment:
There is a flag when generating, `-updateLaunchSettings`, that does this change automatically.